src/OrbitClientModel/CaptureDeserializerTest.cpp

```diff
@@ -207,24 +209,23 @@ TEST(CaptureDeserializer, LoadCaptureInfoOnCaptureStarted) {
   EXPECT_CALL(listener, OnCaptureStarted(_, _, _, _)).Times(0);
   EXPECT_CALL(listener, OnCaptureStarted(_, _, IsEmpty(), _))
       .Times(1)
-      .WillOnce([selected_function, module_info](
+      .WillOnce([&instrumented_function, kInstrumentedFunctionId](
                     ProcessData&& process,
                     absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo>
-                        actual_selected_functions,
+                        actual_instrumented_functions,
```

Comment:
actual means current?

Reply:
actual is used to align with GTEST terminology here. EXPECT_* parameters are named acutal and expected.